### PR TITLE
Create _discord.azhammakestrash.is-a.dev.json

### PR DESCRIPTION
**Pull request by AzhamProdLive**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] You have completed your website. 
- [X] The website is reachable.  
- [X] You're not using Vercel or Netlify.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/_discord.azhammakestrash.is-a.dev.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "AzhamProdLive",
+        "email": "websitedomain.regroup588@passinbox.com"
```
Comment (CuteDog5695) [inline domains/_discord.azhammakestrash.is-a.dev.json:4]: you need to add a comma at the "email" line at the end

File: domains/_discord.azhammakestrash.is-a.dev.json
Comment (CuteDog5695) [inline domains/_discord.azhammakestrash.is-a.dev.json:1]: remove the is-a.dev part in the .json file please

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/_discord.azhammakestrash.is-a.dev.json (hunk 1 above)
Comment (AzhamProdLive) [reply domains/_discord.azhammakestrash.is-a.dev.json]: Done

File: domains/_discord.azhammakestrash.is-a.dev.json
Comment (AzhamProdLive) [reply domains/_discord.azhammakestrash.is-a.dev.json:1]: Done too

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**